src/foam/u2/CurrencyView.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'CurrencyView',
+  extends: 'foam.u2.FloatView',
+
+  documentation: 'View for formatting cents into dollars.',
+
+  axioms: [
+    foam.u2.CSS.create({
+      code: '^:read-only { border: none; background: rgba(0,0,0,0); }'
+    })
+  ],
+
+  properties: [
```

Comment:
You inherit these from FloatView, so you don't need to duplicate. You could just set the precision to 2 though, with:

properties: [ [ 'precision', 2' ] ],